DataFormats/GEMRecHit/src/classes_def.xml

```diff
@@ -36,6 +36,16 @@
   <class name="edm::Wrapper<edm::RangeMap<CSCDetId,edm::OwnVector<GEMCSCSegment,edm::ClonePolicy<GEMCSCSegment> >,edm::ClonePolicy<GEMCSCSegment> > >" splitLevel="0"/>
   <class name="GEMCSCSegmentRef" splitLevel="0"/>
 
+  <class name="ME0Segment" splitLevel="0" ClassVersion="11">
```

Comment:
@Dr15Jones,  do we have some specific recommendations (from the CORE point of view) on splitLevel=0?
I suppose 
- it is a useful setting for objects that make it only to RECO or similar data outputs
- should probably be not set for objects that may go to lighter data tiers (AOD)
- don't make much sense for classes that will never be persisted (e.g. ``` std::vector<ME0RecHit*> ``` )